dyo/dyo.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	cli "github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := &cli.App{
+		Name:      "dyo",
```

Comment:
Naming is still cli, imho.

Reply:
In the repository, the directory/package should be cli, but otherwise the binary and the alias can be **dyo**.

Reply:
I can agree with that.